.github/workflows/greetings.yml

```diff
@@ -0,0 +1,41 @@
+name: Greetings
+
+on:
+- pull_request_target
+# - issues
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        # issue-message: 'not currently posting messages for first interaction via issue'
+        pr-message: >+
+          Thanks for opening your first PR to PlasmaPy!
+
+          Someone should get around to looking into this PR soon. In the meantime,
+          there are a few things you could do:
+
+          * the "Checks" tab at the bottom of the pull request page runs our
+          test suite. Don't worry if something broke! We break stuff all the
+          time. Eventually, we'll want to get tests on this PR to pass before
```

Comment:
This is definitely a good message to convey.  Things probably won't work the first time, but that's totally fine.

Reply:
Yeah; I thought we should have that in here based on Christoph's adventure with the PR template. That one needs changing, too...